templates/docs/examples/patterns/password-toggle/default.html

```diff
@@ -0,0 +1,50 @@
+{% extends "_layouts/examples.html" %}
+{% block title %}Password toggle field{% endblock %}
+
+{% block standalone_css %}patterns_password-toggle{% endblock %}
+
+{% block content %}
+<div class="p-password-toggle">
+  <div class="p-password-toggle__container">
+    <label for="password">Password</label>
+    <button class="p-password-toggle__button" aria-controls="password">
```

Comment:
Accessibility Man strikes again! I think we need assistive technology to announce when this has changed, which we'd get with the `aria-live` attribute:

```suggestion
    <button class="p-password-toggle__button" aria-live="polite" aria-controls="password">
```